config.go

```diff
@@ -382,6 +386,14 @@ func loadAndValidateConfig(interceptor signal.Interceptor) (*Config, error) {
 		return nil, err
 	}
 
+	// To enable the rpc middleware interceptor, we need LND's RPC
+	// middleware interceptor to be enabled. In remote mode we can't
+	// influence whether that's enabled on lnd. But in integrated mode we
+	// can overwrite the flag here.
+	if !cfg.lndRemote {
```

Comment:
I think we shouldn't overwrite the value if the middleware is disabled. Maybe the user wants to have their own interceptor without the manager? So basically move the `!cfg.RPCMiddleware.Disabled` into the `if` statement and then just do `cfg.Lnd.RPCMiddleware.Enable = true`.

Reply:
yeah good point. Fixed!